demo/src/styles.css

```diff
@@ -1 +1,70 @@
 /* You can add global styles to this file, and also import other style files */
+.modal{
```

Comment:
instead of manually copying our base styles into this file, we should use the angular-cli.json 
```
"styles": [
        "styles.css",
        "node_modules/ngx-simple-modal/styles/simple-modal.css"
      ],
```